### PR TITLE
Add multiple emails and verification

### PR DESCRIPTION
**Pull request by auralon**

### CODE REVIEW

**Comment by ottonomy** [comment]
Congratulations on finishing a draft of this code!

File: views/user-profile.html [hunk 1]
```diff
@@ -13,29 +13,207 @@ <h1 class="section-heading">User Profile</h1>
                     </div>
                 </div>
                 <div class="col-xs-12 col-lg-6">
-                    <form action="/user/profile" method="post">
+                    <legend>Name</legend>
+                    <form action="/user/profile/update-name" method="post">
                         <input type="hidden" name="_csrf" value="{{ csrfToken }}">
                         <div class="form-group">
-                              <label>Password</label>
+                            <label>First Name</label>
```
Comment (omidmufeeddm) [inline views/user-profile.html]: Great to see the firstname and lastname are also added in 👍

File: views/user-profile.html [hunk 2]
```diff
@@ -13,29 +13,207 @@ <h1 class="section-heading">User Profile</h1>
...
                         </div>
+                        {% include "includes/password-requirements.html" %}
                         <div class="form-group">
-                            <input class="btn btn-primary btn-outline" type="submit" value="Save">
+                            <input class="btn btn-primary btn-outline password-check" type="submit" value="Update Password" disabled="true">
                         </div>
                     </form>
+
+                    <div class="delete-account push">
+                        <legend>Delete Account</legend>
```
Comment (omidmufeeddm) [inline views/user-profile.html]: Brilliant! It is great that now multiple emails can be combined into a single account and also delete the old accounts.

File: test/backpack-controller.test.js [hunk 3]
```diff
@@ -93,40 +93,40 @@ $.prepareDatabase({
...
-        const expectedImageData = fs.readFileSync(VALID_BAKED_IMAGE).toString('base64');
-        t.same(badges.length, 1);
-        BadgeImage.findOne({badge_hash: badges[0].get('body_hash')}, function (err, image) {
-          t.same(image.get('image_data').toString(), expectedImageData);
-          t.end();
-        });
-      })
-    });
-  });
+  // test('backpack#userBadgeUpload: valid baked badge', function (t) {
```
Comment (omidmufeeddm) [inline test/backpack-controller.test.js]: Looking forward to enabling these back in the next round.

**Pull request merged**